cnnimg.py

```diff
@@ -43,16 +40,16 @@ def __init__(self):
         return
 
     ##Helper Functions start
-    def f(self, x, t):
+    def f(self, x, t, Ib, Bu, tempA):
         x = x.reshape((self.n, self.m))
-        dx = -x + self.Ib + self.Bu + sig.convolve2d(self.cnn(x), self.tempA, 'same')
```

Comment:
Did you test the performance after passing arguments with the ```args``` flag? I was facing some issues with ```args``` flag. I'll also test this again. If this works well, it would remove unnecessary class variables. Good!